### ISSUE TITLE
Cloche can't be extracted from with EIO conduits

### ISSUE THREAD

**Issue by zackeezy** [BluSunrize/ImmersiveEngineering#2317]
https://github.com/SleepyTrousers/EnderIO/issues/4291

IE version: 0.10-61
MC version: 1.10.2
Forge: 12.18.3.2254

**Comment by BluSunrize** [comment]
Pictures.

**Comment by zackeezy** [comment]
Oh sorry. https://imgur.com/a/lvaoc

**Comment by falizure** [comment]
i can confirm this as well, i had to resort to using thermal expansion ducts.

**Comment by HenryLoenwind** [comment]
```
if(dummy==1 && (facing==null||facing==this.facing.getOpposite()))
				return (T)outputHandler;
```

Doesn't this return the outputHandler of the (empty) dummy instead of the TileEntity that has the real data?

**Comment by lstanden** [comment]
Dupe of #2186

**Comment by BluSunrize** [comment]
Hmm, that's a pretty good call @HenryLoenwind 
Dunno how I haven't spotted that before...

**Comment by KingLemming** [comment]
@falizure - "resort to?"

Wow, thanks.

**Comment by falizure** [comment]
sorry, its no offense lemming, your ducts are good at what they do, my goto has always just been conduits because of how they stack in space, i understand yours are getting effects like that but by now its become habit.

**Comment by s4msung** [comment]
Is this fix going to 1.10.2 as well?

**Comment by BluSunrize** [comment]
Nah, probs not.
1.12 is my active version now.

**Comment by s4msung** [comment]
The reason people ask specifically for 1.10 is just the problem that many mods are not in fact updating to 1.11 or 1.12 fast enough, people ask for the next best thing, which holds at least a wider variety of mods to choose from, 1.10.2

**Comment by falizure** [comment]
actually the problem me and my community are having is that the mods for 1.10.2 are SOOO poorly coded...we WANT to move up but...if we go to 1.11 we still don't gain active support for half of them, and in a months time we will be right back to square 1, and if we move to 1.12 we lose half the mods we currently use.

**Comment by KingLemming** [comment]
Move to 1.12. Get over it and thin down the packs. If modders keep supporting 1.10/1.11, people will never move up.

**Comment by mindforger** [comment]
i moved back to 1.7.10 in the meant time and called the inofficial pack "countdown to 1.12" :) feel happy that mods start to pick up 1.12 it can only be a thing of a couple weeks for the first beta and stable releases

**Comment by BluSunrize** [comment]
What Lemming said. 1.7 got people stuck entirely too long. I refuse to support that again.
Move ahead, for your own good, make some new discoveries in variating your packs.

**Comment by s4msung** [comment]
Well, that's annoying. I can't move from 1.10 to 1.12 right now :(
Maybe somewhen in the future

**Comment by falizure** [comment]
I too can not move forward, several mods that have been staples in our packs/servers from 1.6ish have not updated to 1.11 or 1.12 yet, and our community would be very upset if we make packs without them, stuck is a frame of mind, i believe 1.7 was GOOD for modding, because it gave devs time to make their versions stable with almost all possible scenarios where they might be used, their is nothing wrong with new features but please don't drop support for versions that are still practically hot of the presses, minecraft may be rolling out versions faster then the eye can see but that doesn't mean everyone has to follow suit, 1.13 has had its content revealed and 1.14 is already planned, if you continue to only support the newest and latest versions you will lose are large portion of your fan base who are used to playing with more then 1 mod at a time because they cant get matching versions that are all equally supported, i myself am stuck between a rock and a hard place, drop mods that have been stable and fun thus far to regain a measure of support (till the next minecraft update) or continue using mods that are fun together but never have any hope of fixes for anything wrong. i know from a development standpoint maintaining multiple different versions is more work but as a avid fan of not only your work but many others, please don't stop supporting older versions, at least those released within a year or less, its soo hard to make a stable server these days i have had many a sleepless night over the stress its caused trying to explain to players that they are just going to have to live with the issues unless we want a new map every couple months as new versions roll out, with less and less supported mods each time, that isn't to say i wont ever update but until more devs reach a stable build on matching versions i cant push ahaid, this is me practically begging to reconsider. Take that as you will but know any such effort on your part and i would show immense gratitude from mine and likely our community as well.

**Comment by BluSunrize** [comment]
Oh boy, wall of text! 

To be quite frank, there is multiple reasons for me moving ahead. One is the aforementioned "pulling users along", trying to not have them stagnant. 
Others include: not wanting to maintain for multiple versions, and being somewhat haunted by my being stuck on 1.10 for so long (and having my friends have to update for me). I don't want to come to that again.

**Comment by falizure** [comment]
i do apologize for the wall but i wanted to properly explain what big a move this is and its impact on large pack communities, honestly...i could drop 1 or 2 of the mods that havent update without much more then a single tear, but the biggest problem is server support, server tools are not being updated near as fast as even mods are, as it is we have a couple people maintaining our own inhouse punishment plugin just to have something reliable for that, and the people doing so only just have the time to maintain it in its current state, if we move forward we are likely to lose claims, block logging, performance tracking and maybe even reliable economy mods/plugins. and frankly, we rely heavily on forgesponge which as it is is only in beta for 1.10 and anything past that isnt even being taken seriously yet. please understand pushing mods forward is awesome and i love it i love seeing the new content additions but from a server standpoint it is a absolute nightmare that is getting harder and harder as time goes on.

**Comment by xenxero1** [comment]
Would it be a fair compromise to allow people to backport fixes and updates to the older official versions? Minecraft seems to never stop updating, which means that at any given point we wont have all, or atleast all wanted/ideal mods for a pack, on a single supported version. At the very least if the dev doesn't want to support it, let someone else push updates and deal with supporting it.

**Comment by Carnaxus** [comment]
@BluSunrize Are you at least willing to fix repeatable crash issues for the "final" older versions?

**Comment by BluSunrize** [comment]
No, as I have stated many times, I'm not backporting.

**Comment by Carnaxus** [comment]
I didn't say backport, I said fix; I'm stuck with 1.10.2 because the server I'm playing on is 1.10.2, and there's a [crash](https://github.com/BluSunrize/ImmersiveEngineering/issues/2675) happening with either IE, Immersive Petroleum, or from some interaction between the two.  I'm not asking for features from the newer versions to be added to the older version, I'm just asking for a fix other than "lolnope, update or GTFO."

**Comment by BluSunrize** [comment]
I apologize for my misunderstanding, allow me to rephrase:
"I have no intention to release another build for Minecraft 1.10.2"
